iib/web/healthcheck.py

```diff
@@ -0,0 +1,138 @@
+import time
+import logging
+import sqlalchemy
+import kombu
+
+from iib.exceptions import IIBError
+from iib.web import db
+from iib.workers.config import get_worker_config
+from iib.workers.tasks.celery import app
+
+
+log = logging.getLogger(__name__)
+
+
+def database_connection_check():
+    """
+    Check if the database connection is working.
+
+    :return: tuple (ok: bool, reason: str or None)
+    """
+    session = db.session()
+    try:
+        session.execute("SELECT 1")
+    except sqlalchemy.exc.OperationalError as OperError:
+        log.error("database connection failed, exception: %s", OperError)
+        return False, "database connection failed"
+    except sqlalchemy.exc.SQLAlchemyError as err:
+        log.error("an error occurred while trying to connect to the DB server: %s", err)
+        return False, "an error occurred"
+    finally:
+        session.close()
+
+    return True, None
+
+
+def rabbitmq_connection_check(broker_url):
+    """
+    Check if RabbitMQ connection is working.
+
+    :param str broker_url: the AMQP(S) URL to connect to RabbitMQ
+    :return: tuple (ok: bool, reason: str or None)
+    """
+    with kombu.Connection(broker_url) as connection:
+        try:
+            connection.ensure_connection(max_retries=0)
+        except kombu.exceptions.OperationalError as OperError:
+            log.exception("broker connection failed, exception: %s", OperError)
+            return False, "broker connection failed"
+        except kombu.exceptions.KombuError as err:
+            log.error("an error occurred while trying to connect to the RabbitMQ server: %s", err)
```

Comment:
Could you please change double quotes " for single quotes '. In all files. Doc-string is exception.
It is noted in README.md -> `* Single quotes are used instead of double quotes`

Reply:
Don't blame me, I like double quotes but we should keep code style in a project.

Reply:
I've just replaced them! :)